deploy/helm-chart/values.yaml

```diff
@@ -58,6 +58,9 @@ nginx:
     tag: edge
     pullPolicy: Always
 
+  ## The configuration for the data plane that is contained in the NginxProxy resource.
+  config: {}
```

Comment:
To make it easier on users to understand the structure, maybe we add the telemetry config in here but comment it out. Similar to how we did it in the old [NSM chart](https://github.com/nginxinc/nginx-service-mesh/blob/main/helm-chart/values.yaml#L66)